### PR TITLE
jenkins/jobs: Drop i386 images for pld

### PR DESCRIPTION
**Pull request by simondeziel**

### CODE REVIEW

File: jenkins/jobs/image-pld.yaml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-- job:
-    name: "image-pld"
```
Comment (stgraber) [inline jenkins/jobs/image-pld.yaml:2]: What happened here?

Comment (simondeziel) [reply jenkins/jobs/image-pld.yaml]: Whoa, probably bogus keystrokes in vim, fixed now.

File: jenkins/jobs/image-pld.yaml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 - job:
-    name: "image-pld"
-    concurrent: false
+    name: "image-pld"    concurrent: false
```
Comment (stgraber) [inline jenkins/jobs/image-pld.yaml:2]: Still not quite right, you're missing a line break here :)

Comment (simondeziel) [reply jenkins/jobs/image-pld.yaml]: Now the diff looks good.

**Pull request merged**